gn3/api/metadata_api/wiki.py

```diff
@@ -74,13 +74,17 @@ def edit_wiki(comment_id: int):
                                           insert_dict["versionId"], cat_id)
             )
 
-        # Editing RDF:
-        update_wiki_comment(
-            insert_dict=insert_dict,
-            sparql_user=current_app.config["SPARQL_USER"],
-            sparql_password=current_app.config["SPARQL_PASSWORD"],
-            sparql_auth_uri=current_app.config["SPARQL_AUTH_URI"]
-        )
+        try:
+            # Editing RDF:
+            update_wiki_comment(
+                insert_dict=insert_dict,
+                sparql_user=current_app.config["SPARQL_USER"],
+                sparql_password=current_app.config["SPARQL_PASSWORD"],
+                sparql_auth_uri=current_app.config["SPARQL_AUTH_URI"]
+            )
+        except Exception as exc:
+            conn.rollback()
```

Comment:
*issue:*  This raises the following mypy error:

```
gn3/api/metadata_api/wiki.py:86: errogn3/api/metadata_api/wiki.py:86: error: "Connection" has no attribute "rollback"  [attr-defined]r: "Connection" has no attribute "rollback"  [attr-defined]
```
```suggestion
            conn.rollback()  # type: ignore
```

Reply:
Done.